catalog/systemd.pl.catalog.in

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: LGPL-2.1+
 #
-#  Copyright © 2014-2017 Piotr Drąg
+#  Copyright © 2014-2018 Piotr Drąg
```

Comment:
We are trying to drop all copyright lines (on the assumption that `git blame` is more accurate anyway). Please add a comment in #9320 if you are OK or if you are not OK with that for your copyright lines.